js/views/conversation_view.js

```diff
@@ -163,11 +163,7 @@
             this.window.addEventListener('focus', this.onFocus);
 
             extension.windows.onClosed(function () {
-                this.window.removeEventListener('resize', onResize);
-                this.window.removeEventListener('focus', onFocus);
-                window.autosize.destroy(this.$messageField);
-                this.remove();
-                this.model.messageCollection.reset([]);
+                this.unload();
```

Comment:
nit: Might want to update the log statement in `unload` since things can be `unload`ed now *not* just because of "inactivity".